packages/internal/src/cliLogger.ts

```diff
@@ -0,0 +1,20 @@
+import { defaultLoggerOptions } from '@redwoodjs/api/logger'
+
+// Can't use color in the createLogger logger - so use a simpler set of log fns
+const logLevel = defaultLoggerOptions.level
+type CLog = (typeof console)['log']
+
+/**
+ * An alternative to createLogger which supports the same logging levels
+ * but allows for full ANSI when printing to the console.
+ */
+export const cliLogger: CLog & {
+  trace: CLog
+  debug: CLog
+} = function (...data: any[]) {
+  console.log(...data)
+}
+
+cliLogger.trace =
+  logLevel === 'trace' || logLevel === 'debug' ? console.log : () => {}
+cliLogger.debug = logLevel === 'debug' ? console.log : () => {}
```

Comment:
@orta I think the levels here may be swapped, but I'm a little unsure about this so let me write this all out just so I can make sure I have it straight. Here's the pino log levels for reference:

https://github.com/pinojs/pino/blob/802171c88aca42000e88ad1913767975f393cd29/test/transport/core.test.js#L180

Trace has the lowest level, so if I set LOG_LEVEL to trace, I should see debug, info, warn, etc too. But the logic here doesn't show debug if LOG_LEVEL is trace so maybe this is what we want?

```suggestion
cliLogger.trace = logLevel === 'trace' ? console.log : () => {}
cliLogger.debug =
  logLevel === 'debug' || logLevel === 'trace' ? console.log : () => {}
```

Reply:
I think my OP logic is right, trace is lower than debug - this is the psuedo code:

trace = log when trace or debug
debug = log when debug

your version is:

trace = trace
debug = debug or trace

this would make trace show on both! However it is lower, so it should only show on one

Reply:
Resolving this Dom, feel free to unresolve if you have more questions

Reply:
@dac09 did you try this locally? I don't feel like this is resolved based on my understanding of log levels and my experience running it locally:

https://github.com/redwoodjs/redwood/assets/32992335/e3e17c0f-4c64-4a71-a012-5652531f8df6

Reply:
Yeah, you're definitely right in that video 👍🏻